Dockerfile

```diff
@@ -18,12 +18,13 @@ FROM --platform=$BUILDPLATFORM $golang_image AS builder
 WORKDIR /go/src/github.com/kubernetes-sigs/aws-iam-authenticator
 COPY . .
 RUN go mod download
+ARG TARGETOS TARGETARCH
```

Comment:
Can we get a makefile target similar to [build-all-bins](https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/Makefile#L81) for building a set of supported targets?

Reply:
Done